binaries/Readme.md

```diff
@@ -156,11 +156,20 @@ These include Windows and Mac, and multiple variations for Linux distributions.
 
 The binaries are published under the following URL schema.
 
-- Latest: 
+- Latest:
+
```

Comment:
the empty lines in the list item lead to suboptimal formatting on github

Reply:
I do need empty lines though, otherwise everything will be on the same line... if you have a specific formatting which you think would look better, feel free to suggest it, otherwise I don't know how I could improve this

Reply:
Oh, to force line breaks in Markdown you can use `  ` (2 spaces, which is similar to a `<br>` in HTML and GitHub is really bad at rendering here.).

Reply:
So probably e.g.:

```
- Specific commit:  
  `<base url>/<branch>/<commit>/<platform family>/<binary>(.<file extension>).gz`  
  e.g. `https://prisma-builds.s3-eu-west-1.amazonaws.com/master/4028eec09329a14692b13f06581329fddb7b2876/darwin/prisma.gz`
```

to get

- Specific commit:  
  `<base url>/<branch>/<commit>/<platform family>/<binary>(.<file extension>).gz`  
  e.g. `https://prisma-builds.s3-eu-west-1.amazonaws.com/master/4028eec09329a14692b13f06581329fddb7b2876/darwin/prisma.gz`


vs.

- Specific commit:

  `<base url>/<branch>/<commit>/<platform family>/<binary>(.<file extension>).gz`

  e.g. `https://prisma-builds.s3-eu-west-1.amazonaws.com/master/4028eec09329a14692b13f06581329fddb7b2876/darwin/prisma.gz`

Reply:
Ah, thanks. I didn't know the 2 trailing spaces thing, that's a bit weird.. but it works. https://github.com/prisma/specs/pull/355

Reply:
Markdown is weird indeed...